### ISSUE TITLE
Regarding branch names

### ISSUE THREAD

**Issue by ivanhlb** [nus-tic4001-AY2122S1/forum#23]
Might be a small, insignificant question, but I think the names of the branches are a bit redundant? Why `branch-Level-7` and not just `Level-7`?

**Comment by damithc** [comment]
Good point @ivanhlb . The extra word is simply for the ease of distinguishing between the two visually, in tools such as SourceTree that show tags and labels _almost_ the same way.
